x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/tables/meta_engines_table_name_column_content.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiFlexGroup, EuiIcon, EuiHealth, EuiFlexItem } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { EuiLinkTo } from '../../../../../shared/react_router_helpers';
+import { ENGINE_PATH } from '../../../../routes';
+import { generateEncodedPath } from '../../../../utils/encode_path_params';
+import { EngineDetails } from '../../../engine/types';
+
+interface MetaEnginesTableNameContentProps {
+  name: string;
+  isExpanded: boolean;
+  item: EngineDetails;
+  hideRow: (name: string) => void;
+  sendEngineTableLinkClickTelemetry: () => void;
+  showRow: (name: string) => void;
+}
+
+export const MetaEnginesTableNameColumnContent: React.FC<MetaEnginesTableNameContentProps> = ({
+  name,
+  item,
+  isExpanded,
+  sendEngineTableLinkClickTelemetry,
+  hideRow,
+  showRow,
+}) => (
+  <EuiFlexGroup direction="column" gutterSize="none">
+    <EuiLinkTo
+      to={generateEncodedPath(ENGINE_PATH, { engineName: name })}
+      onClick={sendEngineTableLinkClickTelemetry}
+    >
+      <strong>{name}</strong>
+    </EuiLinkTo>
+    <button
+      type="button"
+      onClick={() => (isExpanded ? hideRow(name) : showRow(name))}
+      aria-expanded={isExpanded}
+      className="meta-engines__expand-row"
+    >
+      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+        <EuiFlexItem grow={false}>
+          <EuiIcon type={isExpanded ? 'arrowDown' : 'arrowRight'} />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engines.metaEnginesTable.sourceEnginesCount',
+            {
+              defaultMessage: '{sourceEnginesCount, plural, one {# Engine} other {# Engines}}',
```

Comment:
Minor copy nit - we should be converting old `Title Casing` to `Sentence casing` per EUI guidelines where we can (Chris will do a final pass at some point so no big deal if we miss some, just helps reduce work later)

```suggestion
              defaultMessage: '{sourceEnginesCount, plural, one {# engine} other {# engines}}',
```

Reply:
Thanks for the reminder 😅